common/laws/00_economic_system.txt

```diff
@@ -387,57 +349,48 @@ law_industry_banned = {
 			}
 		}
 	}
-
 	ai_will_do = {
 		always = no
 	}
-
-	ai_enact_weight_modifier = { #Petitions
+	ai_enact_weight_modifier = {
+		#Petitions
 		value = 0
-		
 		if = {
-			limit = { 
+			limit = {
 				has_journal_entry = je_government_petition
 				has_variable = desired_law_var
-                scope:law = var:desired_law_var
+				scope:law = var:desired_law_var
 			}
 			add = 750
 		}
 	}
 }
 
-
 law_laissez_faire = {
 	group = lawgroup_economic_system
-	
 	icon = "gfx/interface/icons/law_icons/laissez_faire.dds"
-	
 	progressiveness = 100
-	
 	disallowing_laws = {
 		law_serfdom
 		law_isolationism
 		law_anarchy
 	}
-	
 	unlocking_technologies = {
 		international_trade
 	}
-	
 	on_activate = {
 	}
-	
 	modifier = {
 		country_loan_interest_rate_mult = -0.15
-		state_shopkeepers_investment_pool_efficiency_mult = 0.25	
-		state_capitalists_investment_pool_efficiency_mult = 0.25
+		state_shopkeepers_investment_pool_efficiency_mult = 0.5
+		state_capitalists_investment_pool_efficiency_mult = 0.3
```

Comment:
I don't like buffing this capitalist efficiency. It's already fairly powerful.

Reply:
Agreed, will look for alternative change